concordia/documents.py

```diff
@@ -21,6 +22,12 @@
 
 @user.doc_type
 class UserDocument(DocType):
+    transcription_count = fields.IntegerField()
+
+    def prepare_transcription_count(self, instance):
+        qs = User.objects.filter(id=instance.id).annotate(Count("transcription"))
+        return qs[0].transcription__count
```

Comment:
If we're only ever getting a single record, should this be 

```python
user = User.objects.annotate(Count("transcription")).get()
return user.transcription__count
```

Reply:
I will fix it. Thx